src/afl-fuzz-one.c

```diff
@@ -453,16 +453,7 @@ u8 fuzz_one_original(afl_state_t *afl) {
 
   }
 
-  /* Map the test case into memory. */
-
-  fd = open(afl->queue_cur->fname, O_RDONLY);
-
-  if (unlikely(fd < 0)) {
-
-    PFATAL("Unable to open '%s'", afl->queue_cur->fname);
-
-  }
-
+  orig_in = in_buf = queue_testcase_get(afl, afl->queue_cur);
   len = afl->queue_cur->len;
 
   orig_in = in_buf = mmap(0, len, PROT_READ | PROT_WRITE, MAP_PRIVATE, fd, 0);
```

Comment:
here are lines that need to be deleted after merge, otherwise it fails. the merge was fucked because of collisions and this one somehow slipped through

Reply:
Heiko fixed it. Dunno I could not see the lines ...